eng/Signing.props

```diff
@@ -1,21 +1,15 @@
 <Project>
 
-  <ItemGroup Condition="'$(PostBuildSign)' != 'true'">
-    <ItemsToSign Remove="@(ItemsToSign)" />
-    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.exe" />
-    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.zip" />
-    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.msi" />
-    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.nupkg" />
-  </ItemGroup>
+  <PropertyGroup>
+    <BlobStorageVersion>$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version'))</BlobStorageVersion>
+    <BlobStorageVersion>$([System.Text.RegularExpressions.Regex]::Split('$(BlobStorageVersion)', '\r\n|\r|\n')[0])</BlobStorageVersion>
```

Comment:
```suggestion
    <BlobStorageVersion>$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version').Trim())</BlobStorageVersion>
```

Reply:
Yeah that works. No idea why I didn't think of trim.